src/grt/src/GlobalRouter.cpp

```diff
@@ -3459,17 +3459,18 @@ class GrouteRenderer : public gui::Renderer
 {
  public:
   GrouteRenderer(GlobalRouter* groute, odb::dbTech* tech);
-  void highlight(const odb::dbNet* net);
+  void highlight(odb::dbNet* net);
+  void clear();
   virtual void drawObjects(gui::Painter& /* painter */) override;
 
  private:
   GlobalRouter* groute_;
   odb::dbTech* tech_;
-  const odb::dbNet* net_;
+  std::vector<odb::dbNet*> nets_;
```

Comment:
is there some reason you can't declare this a vector of const dbNets and leave all of the consts in?